variables.tf

```diff
@@ -7,14 +7,28 @@ variable "create" {
   default     = true
 }
 
+variable "create_sg" {
```

Comment:
I didn't notice before, but you changed the parameter name - it was `create_group`, you changed to `create_sg`. Was there a reason to do that?

Reply:
To be more like the rest of terraform-aws-modules, also "group" is a bit unclear (db_subnet_group in rds/vpc, origin_group in cloudfront).